meta-lmp-base/recipes-sota/aktualizr/aktualizr-pkcs11-label.bb

```diff
@@ -0,0 +1,23 @@
+SUMMARY = "Aktualizr configuration snippet to make sure pkcs#11 token is labeled properly"
+HOMEPAGE = "https://github.com/advancedtelematic/aktualizr"
+SECTION = "base"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MPL-2.0;md5=815ca599c9df247a0c7f619bab123dad"
+
+inherit allarch
+
+SRC_URI = "file://pkcs11-label.toml"
+
```

Comment:
As this recipe don't compile anything, adding `INHIBIT_DEFAULT_DEPS = "1"` is a good ideia to not pull the toolchain on sysroot for building

Reply:
added